### PR TITLE
Highlights fixes

### PR DESCRIPTION
**Pull request by ioanpocol**

### CODE REVIEW

File: client/app/scripts/superdesk-highlights/views/mark_highlights_dropdown_directive.html [hunk 1]
```diff
@@ -1,6 +1,9 @@
-<ul ng-show="highlights">
-	<li ng-repeat="h in highlights"> 
+<ul>
+	<li ng-if="highlights.length" ng-repeat="h in highlights"> 
 	    <button ng-if="is_marked(h)" ng-disabled="true"  option="{{ h.name | uppercase  }}" ng-click="mark_item(h);">{{ h.name | uppercase  }}</button>
 	    <button ng-if="!is_marked(h)" option="{{ h.name | uppercase  }}" ng-click="mark_item(h);">{{ h.name | uppercase  }}</button>
 	</li>
+	<li ng-if="!highlights.length">
+	   <button ng-disabled="true">No available highlights</button>
```
Comment (petrjasek) [inline client/app/scripts/superdesk-highlights/views/mark_highlights_dropdown_directive.html]: it can use `translate` here

**Comment by petrjasek** [comment]
:+1:

File: client/app/scripts/superdesk-highlights/views/mark_highlights_dropdown_directive.html (hunk 1 above)
Comment (ioanpocol) [inline client/app/scripts/superdesk-highlights/views/mark_highlights_dropdown_directive.html]: Added translated but then the text is not shown.
I will add a task for it

**Comment by mugurrus** [comment]
:+1:

**Pull request merged**